.github/workflows/build.yml

```diff
@@ -24,3 +25,18 @@ jobs:
         if: startsWith(github.ref, 'refs/tags/')
         with:
           files: rpms/*.rpm
+      - name: Deploy to nexus
+        uses: Alfresco/alfresco-build-tools/.github/actions/maven-deploy-file@OPSEXP-1693-deploy-file
+        if: startsWith(github.ref_name, 'OPSEXP-1693')
+        with:
+          group-id: org.imagemagick
+          artifact-id: imagemagick-distribution
+          repository-url: https://nexus.alfresco.com/nexus/content/repositories/thirdparty/
+          version: ${{ steps.build.outputs.built-version }}-gha
+          file: rpms/ImageMagick-${{ steps.build.outputs.built-version }}.x86_64.rpm
+          classifier: rocky8
+          files: rpms/ImageMagick-libs-${{ steps.build.outputs.built-version }}.x86_64.rpm
+          classifiers: rocky8-libs
```

Comment:
I think we should use classifiers to reference linux distro and version only. I'm not sure why differenciating libs from other packages can help here

Reply:
I've done this mostly because it's the unique way to deploy multiple files on nexus in a single iteration but I am also facing some inconsistency when using them in the ansible playbook, I think I still need to figure out which is the best way to do that (mostly to avoid breakage in existing playbook)

Reply:
Do you mean `classifiers` and `classifiers` cannot be set to the same value?

Reply:
classifier(s) should be unique with the same set of groupid, artifactid and version

otherwise they will overwrite each other